### DISCUSSION TITLE
v19.16.3-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#134] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.16.3`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.16.3-3.0.1:** (2024-04-27)

- Updated Youtube from `19.14.3` to `19.16.3`
- Updated submodules
  
### Issues in this Release:
- Changing app icons doesn't work 
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*
- Hiding Shorts creates an empty space instead #55

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.16.3-3.0.1'>v19.16.3-3.0.1 - YTLitePlus</a>.</em>

**Comment by CSummerChild** [comment]
I can deal with it for a while, but are you all working on the ad block bug? It's leaving large gaps when scrolling on the main page and causing videos and shorts to disappear. When the ad block is turned off it doesn't do that.

**Reply by Balackburn** [reply]
Please open an issue on the YTLite Github, https://github.com/dayanch96/YTLite

**Comment by TV365-VN** [comment]
Thank you for creating a free YT Plus fake version, but is it safe for my phone? I'm afraid of getting a virus when I install this fake YT Plus on my phone
===
Cảm ơn bạn đã tạo phiên bản giả mạo YT Plus miễn phí nhưng liệu nó có an toàn cho điện thoại của tôi không? Tôi sợ bị nhiễm virus khi cài YT Plus giả này vào điện thoại

**Reply by Balackburn** [reply]
You can build it yourself, and the code is open - if there was any malicious code inside, people would quickly notice, it's safe.

**Reply by TV365-VN** [reply]
Thank you for your contribution, hope you will quickly release the new fake YTB PLUS update according to the original version code released on the App Stos app store

**Comment by arichornlover** [comment]
Hey Balackburn, I see IPAs on this release and one on the previous release, are you sure hosting them is a good idea?

**Reply by Balackburn** [reply]
> Hey Balackburn, I see IPAs on this release and one on the previous release, are you sure hosting them is a good idea?

I forgot to remove them it was to format the Alstore source, will delete later

**Comment by willcj343** [comment]
Hey, love this app and the work you do. I've successfully built the app myself in the past. When I try with the latest version, I get an error. I am including the log of the "workflow" if that helps with anything or shows it's a me problem.
[0_Build YTLitePlus.txt](https://github.com/YTLitePlus/YTLitePlus/files/15214064/0_Build.YTLitePlus.txt)

**Reply by Balackburn** [reply]
Try again it should work

**Reply by willcj343** [reply]
Thanks so much! Worked great.
